lib/class-wp-widget-block.php

```diff
@@ -127,4 +128,12 @@ public function set_is_wide_widget_in_customizer( $is_wide, $widget_id ) {
 		return $is_wide;
 	}
 
+	/**
+	 * Renders the widget form control templates into the DOM.
+	 */
+	public function output_widget_control_templates() {
+		?>
+		<div id="widgets-left"><!-- compatibility with JS which looks for widget templates here --></div>
```

Comment:
Which JS looks for this?

Reply:
None for now, but if we want to support embedding the inserter next to the customizer (like how the old widgets customizer worked), then we probably would need this. We don't need this now though! I can remove it and add back in another PR if we really do need this.

Reply:
Ah gotcha. Yeah let's wait until needed.